alvr/common/src/graphics.rs

```diff
@@ -1,32 +1,19 @@
 use crate::prelude::*;
-use gfx_hal::Instance;
 
-#[cfg(any(windows, target_os = "linux"))]
 pub fn get_gpu_names() -> Vec<String> {
-    #[cfg(windows)]
-    let instance = gfx_backend_dx11::Instance::create("ALVR", 0).unwrap();
-    #[cfg(target_os = "linux")]
-    let instance = gfx_backend_vulkan::Instance::create("ALVR", 0).unwrap();
-    let adapters = instance.enumerate_adapters();
+    let instance = wgpu::Instance::new(wgpu::BackendBit::PRIMARY);
+    let adapters = instance.enumerate_adapters(wgpu::BackendBit::PRIMARY);
```

Comment:
Can you turn these into `VULKAN | DX11 | METAL`? Future wgpu versions might change `PRIMARY`:

>
> **`pub const PRIMARY: BackendBit`**
> All the apis that wgpu offers first tier of support for.
> Vulkan + Metal + DX12 + Browser WebGPU
> **`pub const SECONDARY: BackendBit`**
> All the apis that wgpu offers second tier of support for. These may be unsupported/still experimental.
> OpenGL + DX11

~https://docs.rs/wgpu/0.9.0/wgpu/struct.BackendBit.html

Reply:
It doesn't really matter which backend is used. `wgpu::BackendBit::PRIMARY` is just a default.

Reply:
Indeed but it is like 20 more characters that might prevent a bug.